app/models/group.rb

```diff
@@ -636,11 +636,12 @@ def self.reset_all_counters!
   end
 
   def self.reset_groups_user_count!(only_group_ids: [])
-    where_sql = ""
-
-    if only_group_ids.present?
-      where_sql = "WHERE group_id IN (#{only_group_ids.map(&:to_i).join(",")})"
-    end
+    where_sql =
+      if only_group_ids.present?
+        "WHERE group_id IN (#{only_group_ids.map(&:to_i).join(",")}) AND user_id > 0"
+      else
+        "WHERE user_id > 0"
+      end
 
     DB.exec <<-SQL
       WITH X AS (
```

Comment:
Would be nice at the same time to rename this from `X` to something sensible like `group_user_counts` :) Less references to Musk the better :P